packages/frontend/editor-ui/src/types/telemetry.ts

```diff
@@ -0,0 +1,13 @@
+export type TelemetryNdvType = 'ndv' | 'focus_panel' | 'zoomed_view';
+
+export type TelemetryNdvSource =
+	| 'added_new_node'
+	| 'canvas_default_view'
+	| 'canvas_zoomed_view'
+	| 'focus_panel'
+	| 'logs_view'
+	| 'other';
+
+export interface TelemetryContext {
+	view_shown: TelemetryNdvType;
```

Comment:
I would like this composable to be as reusable as possible. The naming `view_shown` kind of suggests that you indicate what route you're on. 

Can we rename it to something like `{ view_shown: 'ndv', ndv_source: TelemetryNdvType }` or just `{ ndv_source: TelemetryNdvType }` instead?

Reply:
Interesting idea, I updated like [e292ec1](https://github.com/n8n-io/n8n/pull/18871/commits/e292ec1a9c40e69ae7d74e2279fca25f156f6237), what do you think? Note that we still need `lastSetActiveNodeSource` in NDV store because `useTelemetryContext` is essentially just an inject/provide.